server/src-lib/Hasura/RQL/DDL/Remote/Input.hs

```diff
@@ -0,0 +1,137 @@
+{-# LANGUAGE NamedFieldPuns #-}
```

Comment:
Can we use `Hasura.GraphQL.Validate.InputValue` for these types?

Reply:
What would you re-use from that module?

Reply:
I was under the assumption that parsing the input arguments of a field is already available in that module, considering we are doing the validation of the input arguments for a hasura query.